### ISSUE TITLE
getDataChampion returning 403

### ISSUE THREAD

**Issue by AleTassi** [taycaldwell/riot-api-java#148]
Hi everyone
I'm trying to get the name of a champion from the id. Here's my code

	public String getPlayedChampion(Summoner player) {
		
		try {
			CurrentGameInfo game = api.getActiveGameBySummoner(Platform.EUW, player.getId());
			
			CurrentGameParticipant participant = game.getParticipantByParticipantId(player.getId());
			
			Champion champ = api.getDataChampion(Platform.EUW, participant.getChampionId());
			
			return champ.getName();
		} catch (RiotApiException e) {
			log.warn("Error getPlayedChampion: {}", e.getMessage());
			return null;
		}
		
	}

but the code always throw an error (403 - Forbidden).
I can't find the endPoint that the library try to call in the [Riot Dev Site](https://developer.riotgames.com/api-methods/#champion-v3)
How can i get the name of a champion?
Is there another java method/riot endpoint to get the name?
Thank you 😄

**Comment by KaluNight** [comment]
This endpoint has been depracated and no usable. Now you need to use ddragon (https://ddragon.leagueoflegends.com/cdn/dragontail-9.7.1.tgz). Contains all the assets of the game. You can load all the champions you need in the JSON file named "champion.json" with a library like Gson (https://github.com/google/gson).

**Comment by Pankraz01** [comment]
Yes this means, your api key is invalid. I had this a moment ago, so I need to create a new key...
